web/src/main/java/org/openmrs/web/filter/util/FilterUtil.java

```diff
@@ -104,6 +102,14 @@ public static String restoreLocale(String username) {
 						log.debug(DATABASE_CLOSING_ERROR, e);
 					}
 				}
+
+				if (results != null) {
+					try {
+						results.close();
+					} catch (SQLException e) {
+						log.warn("Error while closing ResultSet");
```

Comment:
Shouldn't you add the exception as the second argument in log.warn?

Reply:
Done :)